perceval/utils.py

```diff
@@ -44,6 +44,9 @@
 
 DEFAULT_DATETIME = datetime.datetime(1970, 1, 1, 0, 0, 0,
                                      tzinfo=dateutil.tz.tzutc())
+LAST_DATETIME = datetime.datetime(2050, 1, 1, 0, 0, 0,
+                                  tzinfo=dateutil.tz.tzutc())
+CURRENT_DATETIME = datetime.datetime.now(dateutil.tz.tzutc())
```

Comment:
This approach will generate unexpected results when `perceval` is used as a library. This date will be generated when the module is loaded in Python. That means it will always use the same date in long running processes. For instance, in dameons.

`DEFAULT_LAST_DATE` should be a better solution but I would set it to `2100-01-01`, as we have in other places.